lib/plugins/css_plugin.js

```diff
@@ -32,9 +32,9 @@ export default class CSSWebpackPlugin {
         const srcFn = dep._source._value
         let src = _eval(srcFn, f) // eslint-disable-line
         src = src[0][1] // this part is questionable, but it is what it is
-
         const outputPath = util.getOutputPath(compiler, f, this.opts)
-        compilation.assets[outputPath] = {
+        const newPath = outputPath.replace(/\.[^/.]+$/, '.css')
```

Comment:
you mean as opposed to declaring a `const`? yeah, i had it in there as first, but since we were using such a brute way of renaming, i thought it'd be best to have it happen on it's own line.